### ISSUE TITLE
Logging in as an existing user without a social credential updates password

### ISSUE THREAD

**Issue by deevus** [GeneaLabs/laravel-socialiter#8]
If you have an existing user who has not logged in using socialiter, their password will get changed by socialiter on their first social login. 

https://github.com/GeneaLabs/laravel-socialiter/blob/a4c83d818f2a12c1906a2c5ed2744a2cd784f5d8/src/Socialiter.php#L75-L86

Should it only set a `password` if the user does not exist?

**Comment by mikebronner** [comment]
@deevus Good point! Thanks for bringing this up. This has not been an issue yet in my experience, but I will think about this for the next update.
